packages/rancher-istio/charts/configs/istio-base.yaml

```diff
@@ -57,6 +65,9 @@ spec:
         name: istio-ingressgateway
         type: {{ .Values.ingressGateways.type }}
     global:
+      {{- if .Values.global.istioNamespace }}
+      istioNamespace: {{ .Values.global.istioNamespace }}
+      {{- end }}
```

Comment:
Could we use `{{ .Values.global.istioNamespace | default "cattle-istio-system" }}` here?

Reply:
the reason I opted not to is if we aren't changing the namespace we don't need to set this value at all as istio will deploy in the istio-system namespace.

Reply:
I take that back, since we are using this flag in installer image, we will want to ensure its always set - I went ahead to updated it with your suggestion

Reply:
we may want to update this based on the resolution in https://github.com/rancher/charts/pull/531#discussion_r469574900